roles/install_dbserver/tasks/install_dbserver.yml

```diff
@@ -31,5 +31,9 @@
     - force_install is defined
     - force_install
 
+- name: Update packages
+  package: name=* state=present
+  become: yes
+
```

Comment:
Updating all packages will dramatically increase deployment time. Do we really need to do that?